src/javascript/binary/websocket_pages/cashier/payment_agent_list.js

```diff
@@ -1,31 +1,30 @@
-const showLoadingImage = require('../../base/utility').showLoadingImage;
-const Cookies          = require('../../../lib/js-cookie');
 const Content          = require('../../common_functions/content').Content;
+const Cookies          = require('../../../lib/js-cookie');
```

Comment:
I think Cookies can move to the last line since it's three folders up, @binary-hamedanchi is that right?